about/about-us.html

```diff
@@ -20,6 +20,7 @@ <h2 class="sub-heading">Our Robotics Competitions</h2>
 		</div>
 		<div id="fall-image" class="image-col"></div>
 	</div>
+	<!--
```

Comment:
Is there a reason you took out the spring section? I think we should keep this on there, as it's helpful for schools searching for information about our spring program